barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/protogalaxy_recursive_verifier.test.cpp

```diff
@@ -7,8 +7,10 @@
 #include "barretenberg/protogalaxy/protogalaxy_verifier.hpp"
 #include "barretenberg/stdlib/hash/blake3s/blake3s.hpp"
 #include "barretenberg/stdlib/hash/pedersen/pedersen.hpp"
+#include "barretenberg/stdlib/hash/sha256/sha256.hpp"
```

Comment:
Are these additional headers needed? I haven't seen sha or packed_byte_array being used in the changes

Reply:
wops leftovers, will remove, thanks